unstructured/ingest/connector/registry.py

```diff
@@ -22,6 +22,7 @@
 from unstructured.ingest.connector.hubspot import HubSpotIngestDoc
 from unstructured.ingest.connector.jira import JiraIngestDoc
 from unstructured.ingest.connector.local import LocalIngestDoc
+from unstructured.ingest.connector.mongodb import MongoDBIngestDoc, MongoDBIngestDocBatch
```

Comment:
not a big deal, but do we actually need MongoDBIngestDoc in the registry? it's only ever used internally by MongoDBIngestDocBatch, right? ...i.e. it won't need to be serialized/deserialized?

Reply:
Seems as though it needs both for:

`create_ingest_doc_from_dict`

Reply:
ahh, forgot that we now serialize between each stage. yep, makes sense.